sos/utilities.py

```diff
@@ -275,7 +275,8 @@ def _check_poller(proc):
         else:
             expanded_args.append(arg)
     if to_file:
-        _output = open(to_file, 'w')  # pylint: disable=consider-using-with
+        # pylint: disable=consider-using-with
```

Comment:
This diable shouldn't be needed since we're now specifying the encoding, right?

Reply:
That's a different warning (`consider-using-with`).

Reply:
Oh, duh. Sorry, went cross-eyed there I guess.